### PR TITLE
Check coverage

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: check_coverage.bat [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#Ubuntu
+#MIN_LINES_COV=74.9
+#MIN_FUNCTION_COV=86.5
```
Comment (mzegla) [inline check_coverage.bat:5]: Why these lines are commented, but RHELs aren't?

File: Dockerfile.redhat [hunk 2]
```diff
@@ -179,6 +183,14 @@ RUN cd /ovms/src/example/SampleCpuExtension/ && make
 
 RUN bazel test ${debug_bazel_flags} --jobs $JOBS --test_summary=detailed --test_output=streamed  //src:ovms_test
 
+COPY check_coverage.bat /ovms/
+# Test Coverage
+RUN if [ "$CHECK_COVERAGE" == "1" ] ; then true ; else exit 0 ; fi ; wget https://github.com/linux-test-project/lcov/releases/download/v1.16/lcov-1.16-1.noarch.rpm &&\
+    yum install -y lcov-1.16-1.noarch.rpm &&\
+    bazel coverage --combined_report=lcov //src:ovms_test &&\
+    genhtml --output genhtml "$(bazel info output_path)/_coverage/_coverage_report.dat" &&\
+    ./check_coverage.bat && exit $?
```
Comment (mzegla) [inline Dockerfile.redhat:192]: I guess it not possible to run coverage check before main building process, so we could fail faster?

Comment (rasapala) [reply Dockerfile.redhat:192]: I can try it.

File: check_coverage.bat (hunk 1 above)
Comment (rasapala) [reply check_coverage.bat:5]: The rhel ones are lower thus we can use it. Currently onCommit tests rhel.

File: Dockerfile.redhat (hunk 2 above)
Comment (rasapala) [reply Dockerfile.redhat:185]: Done.

**Review by dtrawins — APPROVED** [review]

**Review by mzegla — APPROVED** [review]

**Pull request merged**